open-hackathon-server/src/hackathon/user/user_manager.py

```diff
@@ -197,48 +193,25 @@ def user_display_info(self, user):
         if user is None:
             return None
 
-        ret = {
-            "id": user.id,
-            "name": user.name,
-            "nickname": user.nickname,
-            "email": [e.dic() for e in user.emails.all()],
-            "provider": user.provider,
-            "avatar_url": user.avatar_url,
-            "online": user.online,
-            "create_time": str(user.create_time),
-            "last_login_time": str(user.last_login_time),
-            "is_super": self.is_super_admin(user)
-        }
-        if user.profile:
-            ret["user_profile"] = user.profile.dic()
-
-        return ret
-
-    def is_super_admin(self, user):
-        """Check whether an user is super admin or not
-
-        super admin is also known as system administrator who have the highest privileges
-
-        :type user: User
-        :param user: User instance to be returned which shouldn't be None
+        ret = user.dic()
 
-        :rtype bool
-        :return True if user is super admin otherwise False
-        """
-        if user.id == ReservedUser.DefaultSuperAdmin:
-            return True
+        # to fit old return value's fields
+        if "emails" in ret:
+            ret["email"] = ret.pop("emails")
+        if "profile" in ret:
+            ret["user_profile"] = ret.pop("profile")
```

Comment:
We don't need convert `profile` to `user_profile`, keep it the same as DB. Update the client side instead. Similar for `emails`.